module-4/backend/week_one.md

```diff
@@ -5,18 +5,59 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
+  I learned about unit testing with mocha and chai. 
+
 2. What are some tools to help debug JavaScript code?
+
+  The Chrome JavaScript developer tools w/debugger in the code, repl.it for JavaScript & pry for JavaScript.2. What are some tools to help debug JavaScript code?
+  
 3. What are some tools you need in order to unit test your JavaScript?
+
+  Mocha and Chai
+  
 4. What is the syntax for invoking a function?
+
+  functionName()
+  
 5. What is CORS?
+
+  CORS is Cross Origin Resource Sourcing which allows for cross-domain communication from the browser. It allows for domains to access data from each other.
+  
 6. How do we enable CORS?
+
+  CORS is enabled within access headers.
+  
 7. What's `this` in JavaScript?
+
+  'this' is a JS keyword which allows for a temporary value for the methods and properties of an object to be used in a function without needing to repeat the object.
+  
```

Comment:
I would also mention that the value of `this` changes in the context it is used in. In the global scope, it refers to your browser's window. In a constructor function, it would refer to the object declared, etc.